docs/configure-artifact-repository.md

```diff
@@ -360,36 +360,36 @@ artifacts:
       container: my-container-name
       blob: path/in/container
       # If a managed identity has been assigned to the machines running the
-      # workflow (e.g., https://docs.microsoft.com/en-us/azure/aks/use-managed-identity)
+      # workflow (for example, https://docs.microsoft.com/en-us/azure/aks/use-managed-identity)
       # then useSDKCreds should be set to true. The accountKeySecret is not required
       # and will not be used in this case.
       useSDKCreds: true  
 ```
 
-In addition to managed identities, Argo workflows also support authentication using access keys and SAS tokens.
+You can also authenticate with [Access Keys](#using-azure-access-keys) and [Shared Access Signatures (SAS)](#using-azure-shared-access-signatures-sas).
 
-### Using Azure access keys
+### Using Azure Access Keys
 
-1. Retrieve the blob service endpoint for the storage account. For example:
+1. Retrieve the blob service endpoint for the storage account.
 
     ```bash
     az storage account show -n mystorageaccountname --query 'primaryEndpoints.blob' -otsv
     ```
 
-2. Retrieve the access key for the storage account. For example:
+2. Retrieve the Access Key for the storage account.
 
     ```bash
     az storage account keys list -n mystorageaccountname --query '[0].value' -otsv
     ```
 
-3. Create a Kubernetes secret to hold the storage account key. For example:
+3. Create a Kubernetes secret to hold the storage account key.
 
     ```bash
     kubectl create secret generic my-azure-storage-credentials \
       --from-literal "account-access-key=$(az storage account keys list -n mystorageaccountname --query '[0].value' -otsv)"
```

Comment:
```suggestion
      --from-literal "account-access-key=$ACCESS_KEY"
```

then can use this env var here

Reply:
I'll add a line above showing how to get the `ACCESS_KEY` as well 👍

Reply:
The line above isn't necessary after https://github.com/argoproj/argo-workflows/pull/13376#discussion_r1687076355